src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -38,6 +38,9 @@ paths:
   /person:
     $ref: 'paths/person.yaml'
 
+  /person/register:
```

Comment:
@danidoni the path is `/person?cmd=register`

Reply:
the parameter also need to be added to the `person_register.yaml`

Reply:
I'm talking about [this](https://github.com/openSUSE/open-build-service/blob/master/src/api/config/routes/api_routes.rb#L23) one...

Reply:
ohh sorry for the noise, overlooked that there are two endpoint for the same thing.....